shuttle/templates/index.html

```diff
@@ -1,5 +1,56 @@
 {% extends 'shuttle_base.html' %}
 
 {% block body_content %}
-    <h1>Hello World</h1>
+
+    <div class="center-page">
+        <div class="align-side-by-side">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Check In</h4><hr>
+                    <p class="card-text">The Shuttle was last checked in at: </p>
+                    <div id="last-check-in"><h5></h5></div><hr>
+                    <p class="card-text">At the time of: </p>
+                    <div id="last-time-in"><h5></h5></div><hr>
+                    <p>*Based on the Driver's last check in*</p>
+                </div>
+            </div>
+        </div>
+        <div class="align-side-by-side">
+            <div class="card">
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Current Route</h4><hr>
+                    <p class="card-text">The Shuttle is in route to: </p>
+                    <div id="next-check-in"><h5></h5></div><hr>
+                    <p class="card-text">And should arrive at: </p>
+                    <div id="next-time-in"><h5></h5></div><hr>
+                    <p>*Based on the Driver's last check in<br>and the Shuttle schedule*</p>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        $(document).ready(function() {
+            window.onload = check_in();
+            window.onload = next_stop();
+            function check_in() {
+                $.ajax({
+                    url: "{{ url_for('SchedulesView:grab_recent_driver_data') }}",
+                })
+                    .done(function (data) {
+                        $("#last-check-in h5").html(data['location']);
+                        $("#last-time-in h5").html(data['time']);
+                    });
+            }
+            function next_stop() {
+                $.ajax({
+                    url: "{{ url_for('SchedulesView:grab_current_route') }}",
+                })
+                    .done(function (data) {
+                        $("#next-check-in h5").html(data['location']);
+                        $("#next-time-in h5").html(data['time']);
+                    });
+            }
+        });
```

Comment:
Since these two functions seem to be called every time the page is loaded couldn't you just load in the data from python at the same time that you render the template?